Sources/PointFree/Auth.swift

```diff
@@ -79,42 +79,72 @@ private func readGitHubSessionCookieMiddleware(
 }
 
 private func writeGitHubSessionCookieMiddleware(
-  _ conn: Conn<HeadersOpen, Either<Prelude.Unit, GitHubUserEnvelope>>
+  _ conn: Conn<HeadersOpen, GitHubUserEnvelope>
   )
-  -> IO<Conn<HeadersOpen, Either<Prelude.Unit, GitHubUserEnvelope>>> {
```

Comment:
this didn't need to be mapping into `IO<Conn<HeadersOpen, Either<Prelude.Unit, GitHubUserEnvelope>>>` so I just fixed it...